sdk/identity/azure-identity/TROUBLESHOOTING.md

```diff
@@ -294,7 +294,13 @@ Get-AzAccessToken -ResourceUrl "https://management.core.windows.net"
 |---|---|---|
 |The current credential is not configured to acquire tokens for tenant <tenant ID>|<p>The application must configure the credential to allow token acquisition from the requested tenant.|Make one of the following changes in your app:<ul><li>Add the requested tenant ID to `additionally_allowed_tenants` on the credential options.</li><li>Add `*` to `additionally_allowed_tenants` to allow token acquisition for any tenant.</li></ul></p><p>This exception was added as part of a breaking change to multi-tenant authentication in version 1.11.0. Users experiencing this error after upgrading can find details on the change and migration in [BREAKING_CHANGES.md](https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/identity/azure-identity/BREAKING_CHANGES.md#1110).</p> |
 
-## Troubleshoot Web Account Manager (WAM) and Microsoft account (MSA) login issues
+## Troubleshoot Web Account Manager (WAM) brokered authentication issues
+
+| Error Message |Description| Mitigation |
+|---|---|---|
+|AADSTS50011|The redirect URI specified in the request does not match the redirect URIs configured for the application.|Ensure that one of redirect URIs registered for the Microsoft Entra application matches the following URI pattern: `ms-appx-web://Microsoft.AAD.BrokerPlugin/{client_id}`|
```

Comment:
The redirect URL in the request is not specified by the user. User can only specify the port. No?

Reply:
This is an error returned by MS Entra ID if a user authenticates with a custom client/tenant ID, and the corresponding Entra application doesn't have `ms-appx-web://Microsoft.AAD.BrokerPlugin/{client_id}` registered. If anything is passed in to our credential via the `redirect_uri` kwarg, then this is ignored since the WAM URI is very specific (i.e. needs to be that value).

Reply:
We don't us entire `redirect_uri`.

If user specifies it, we only parse it to get the port information and pass the port information to msal.

https://learn.microsoft.com/en-us/entra/identity-platform/scenario-desktop-acquire-token-wam#redirect-uri

Reply:
MSAL doesn't use the port information we pass to it when the broker is enabled. I feel like we should consider removing the `redirect_uri` docstring from `InteractiveBrowserBrokerCredential` since, ultimately, the redirect URI used/expected will be `ms-appx-web://Microsoft.AAD.BrokerPlugin/{client_id}`.

Reply:
No. We cannot remove it. Because:
1. This will be a breaking change.
2. We still need to support the non-broker scenario and confidential app scenario.